Basic Rules.txt

```diff
@@ -117,9 +115,14 @@ Class 		- Think of this like your job. Are you a sniper, waiting for the
 	Classes define what armor and weapons your character can use in combat,
 	as well as certain special abilities which might help solve puzzles or
 	get you better discounts at stores. Your class also determines how much 
-	money you start with to buy equipment and how many nanites you can use
+	money you start with to buy equipment and how many Nanites you can use
 	at once.
-Nanites  	- 
+Nanites  	- The evolution of technology brought about the ability to design
+	and control microscopic machines. Such a powerful advancement assists in
+	all forms of life in the Compound X universe. Nanites are used to activate
+	abilities that are not normally possible on Earth. Abilities such as sprinting 
+	extremely fast, aiming incredibly well, infusing bullets with electrical
+	energy, and more.

```

Comment:
While I'm normally quite loathe to accept scope creep, I think you handled it pretty well here. I do think "not normally possible _on earth_ " is a little non-diajetic and I'd like to see if we can tweak it, but that's a nitpick. Great examples!

Reply:
I'll try and change it to something more in-game. I noticed that other areas seemed to break some of the '4th wall' if you will, so I was trying to build some excitement by comparing what's possible on earth.